checks/apps/deeplearning/pytorch/check_pytorch_dist.py

```diff
@@ -0,0 +1,116 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class pytorch_distr_cnn_base(rfm.RunOnlyRegressionTest):
+    valid_systems = ['lumi:gpu']
+    valid_prog_environs = ['builtin']
+    sourcesdir = 'src'
+    num_tasks = 32
+    num_tasks_per_node = 8
+    num_gpus_per_node = 8
+    variables = {
+        'NCCL_DEBUG': 'INFO',
+        'NCCL_SOCKET_IFNAME': 'hsn0',
```

Comment:
I assume there are 8 gpus per node in use, shouldn't be also all 4 nics (hsn0...3) involved?